packages/es-components/webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  mode: 'production',
+  output: {
+    path: path.join(__dirname, 'bundle'),
+    filename: "[name].min.js"
```

Comment:
Are we sure we are compiling into a umd bundle correctly? I looked at `bundle/main.min.js` and I'm seeing the `!function` declaration common to IIFE.